packages/interbit-covenant-tools/src/tests/coreCovenant/revokeWriteJoin.test.js

```diff
@@ -0,0 +1,188 @@
+const assert = require('assert')
+const Immutable = require('seamless-immutable')
+
+const { PATHS } = require('../../coreCovenant/constants')
+const {
+  consuming: getConsuming,
+  providing: getProviding,
+  roles: getRoles,
+  actionPermissions: getActionPermissions
+} = require('../../coreCovenant/selectors')
+const {
+  revokeReceiveActions,
+  revokeSendActions
+} = require('../../coreCovenant/helpers/revokeJoin')
+
+const CHAIN_ID =
+  '4a733bc88e40ca97073a8620abaad200318ae11ea7312e5f41eae3c36c56f551'
+const ACTION_TYPE = 'underTheHood/spoke/MOUNT_TOKEN'
+
+const initialState = Immutable.from({
+  interbit: {
+    config: {
+      consensus: 'PoA',
+      providing: [
+        {
+          consumer:
+            '4a733bc88e40ca97073a8620abaad200318ae11ea7312e5f41eae3c36c56f551',
+          statePath: [
```

Comment:
It may make sense to use constants for these chain IDs. It may make the tests easier to read.